Automattic/jetpack
# Boost: Remove woocommerce script exclusion from minify JS

## Proposed changes:
* Reverts #37403 as we have done #37728

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
* Install Boost and WooCommerce 8.9
* Enable Concatenate JS on Boost
* Go to the Cart page
* The cart page should load properly
